script/tool/lib/src/create_all_packages_app_command.dart

```diff
@@ -229,18 +235,28 @@ dependencies {}
 ''');
     }
 
-    const String lifecycleDependency =
-        "    implementation 'androidx.lifecycle:lifecycle-runtime:2.2.0-rc01'";
+    final String lifecycleDependency = gradleFileIsKotlin
+        ? '    implementation("androidx.lifecycle:lifecycle-runtime:2.2.0-rc01")'
+        : "    implementation 'androidx.lifecycle:lifecycle-runtime:2.2.0-rc01'";
 
     _adjustFile(
       gradleFile,
       replacements: <String, List<String>>{
-        // minSdkVersion 21 is required by camera_android.
-        'minSdkVersion': <String>['minSdkVersion 21'],
-        'compileSdkVersion': <String>['compileSdk 34'],
+        if (gradleFileIsKotlin)
+          'compileSdk': <String>['compileSdk = 34']
+        else ...<String, List<String>>{
+          // minSdkVersion 21 is required by camera_android.
+          'minSdkVersion': <String>['minSdkVersion 21'],
+          'compileSdkVersion': <String>['compileSdk 34'],
+        }
       },
       additions: <String, List<String>>{
-        'defaultConfig {': <String>['        multiDexEnabled true'],
+        'defaultConfig {': <String>[
+          if (gradleFileIsKotlin)
+            '        multiDexEnabled = true'
+          else
+            '        multiDexEnabled true'
+        ],
```

Comment:
I think we can remove this due to the `minSdkVersion` being 21, but I'll look into that as a follow-up.